### DISCUSSION TITLE
Accepting wildcards when filtering OPERA Burst ID?

### DISCUSSION THREAD

**Discussion by scottstanie** [asfadmin/Discovery-asf_search#257] [Ideas]
Hi, I have two questions that came up while searching the OPERA-S1 datasets:

https://search.asf.alaska.edu/#/?maxResults=250&dataset=OPERA-S1&end=2023-01-02T04:59:59Z&productTypes=CSLC&resultsLoaded=true&zoom=7.353&center=-120.311,34.035&granule=OPERA_L2_CSLC-S1_T144-308014-IW2_20230101T140033Z_20230929T090842Z_S1A_VV_v1.0&start=2023-01-01T05:00:00Z&operaBurstID=T144_308014_IW2

1. Is it possible to relax the "Burst S1 Filter" to be insensitive to `-` or `_` separators? I noticed that if you copy-paste the burst ID within a filename into the filter field (e.g. `T144-308014-IW2` above), it doesn't match because there are dashes instead of underscores. It does already allow lower or upper case, but I'm not sure if it's filtering with a regex (e.g. https://github.com/opera-adt/opera-utils/blob/main/src/opera_utils/constants.py#L21-L24 ), or whether this goes into the CMR query and it's harder to do

2. Is there any easy way to add wildcard capabilities to the Burst ID filter? Some uses would be getting the 3 consecutive bursts along track (`T144-308014-IW*`) or limiting a search to one track like you can with sentinel-1 SLCs (`T144*`).

**Comment by asjohnston-asf** [comment]
+1 for being able to filter the OPERA-S1 and S1 Bursts data sets by path/track number. I've wanted it as a quick way to show "here's what some S1 bursts looks like", as well as to find RTCs from a common pass that would be candidates for mosaicking.
